lib/bulk_replace.rb

```diff
@@ -14,6 +14,12 @@ def bulk_replace(github_token:, file_path:, old_content:, new_content:, global:,
     quit_requested = true
   end
 
+  puts "Search for references of '#{old_content}' and replace with '#{new_content}' in the following file:"
+  puts "#{file_path}"
+  printf "\e[31mPress 'y' to continue: \e[0m"
+  prompt = $stdin.gets.chomp
+  exit 130 unless prompt == "y"
```

Comment:
Quick flyby comment - this looks like the wrong exit code to me - [130 should be used if the script was interrupted](https://unix.stackexchange.com/questions/386836/why-is-doing-an-exit-130-is-not-the-same-as-dying-of-sigint) e.g. via CTRL + C)

Reply:
Agree, I think exit code 0 makes sense here since choosing not to continue the script isn't really an error per se